app/particle/Particle.py

```diff
@@ -69,20 +71,16 @@ def update_velocity(self, particles):
         neighbors = []
         for particle in particles:
             euclidean_distance, distances = self.euclidean_distance_to(particle)
-            #if euclidean_distance <= 0 and particle is not self:
             if euclidean_distance <= 5*(self.radius + particle.radius) and particle is not self:
                 neighbors.append((particle, distances, euclidean_distance))
 
-        #p_energy = 0
         max_force = 5000
         for neighbor, distances, euclidean_distance in neighbors:
-            #p_energy = p_energy + 5000 *( ( 1 / euclidean_distance ) - (1 / (5*(self.radius + particle.radius) ) )
             for i in range(3):
-                #max_force = 10
                 if euclidean_distance == 0:
                     force = max_force
                 else:
-                    force = params.force * (1 * (distances[i])/euclidean_distance**3)/self.mass
+                    force = params.force * distances[i] / euclidean_distance**3 / self.mass
```

Comment:
i forgot to do that before i pushed the changes to my fork, but i will make the changes in the next push!